### PR TITLE
feat(content): added i18n content

### LINKED ISSUE #151: Add Korean json file
``` 
{
  "contact": "연락",
  "copyright": "저작권",
  "support": "지원",
  "es": "스페인어",
  "en": "영어",
  "ko": "한국어",
  "cn": "중국어",
  "totalProxiedVotes": "총 프록시 투표 수",
  "routes": {
    "/": "메인",
    "voters": "회원",
    "vote": "투표",
    "about": "소개",
    "mustLogin": "로그인해야 합니다.",
    "successfulVote": "투표 완료",
    "somethingWrong": "문제가 발생했습니다. 다시 시도해주세요.",
    "signin": "로그인",
    "logout": "로그아웃"
  },
  "about": {
    "votersMetaTitle": "소개",
    "whatIsEden": "에덴 프록시 소개",
    "intention": "의도",
    "intentionDescription": "에덴 프록시는 에덴의 선거 과정에서 도출된 합의의 정당성뿐만 아니라 에덴 커뮤니티가 신원을 검증한 EOS 보유자들의 책임과 평판을 활용하여 EOS 거버넌스 책임을 극대화하기 위한 의도로 출범하였습니다.",
    "edenDescription": "에덴 스마트 프록시는 EOS 블록 생산자(BP) 투표를 통하여 에덴 선거 절차에 따라 선출된 대표자와 에덴 커뮤니티 회원에게 EOS 거버넌스에 대한 영향력을 높일 수 있는 권한을 부여합니다.",
    "howDoesWork": "작동 방식",
    "howDoesWorkDescription": "에덴 스마트 프록시는 에덴 커뮤니티의 목소리를 반영할 것입니다. 대표자들의 투표 파워는 각 선거에 참여한 참여자 수에 따라 달라집니다. 예를 들어 다음과 같습니다.",
    "point1": "총 참여자 / L1 대표자 = L1 대표자 가중치",
    "point2": "총 참여자 / L2 대표자 = L2 대표자 가중치",
    "note": "참고:",
    "notePointer1": "이전 선거에 참여한 에덴 커뮤니티 회원들에게만 투표 가중치가 반영됩니다.",
    "notePointer2": "이전 대표자는 현재 라운드에서는 더 이상 대표자가 아니므로 투표 가중치는 없어집니다.",
    "notePointer3": "총 참여자는 선거에 참여한 에덴 회원 수입니다.",
    "notePointer4": "수석 L2 대표자는 다른 L2 대표자들과 동일한 투표 가중치를 갖습니다.",
    "notePointer5": "투표 가중치는 정수로 반올림됩니다.",
    "exampleCalculations": "계산 예시",
    "exampleDescription": "4차 에덴 선거 결과:",
    "exampleDescription2": "에덴 스마트 프록시는 가장 최근 에덴 선거 참여자와 선출된 대표자들의 집단적 의사 결정을 반영하는 집계 점수 순으로 최대 30 팀의 BP에게 투표합니다.",
    "exampleDescription3": "모든 긴급 상황(예: 지속적으로 블록 생산을 누락되거나 중요한 의사 결정에 무응답 또는 부재 등)의 경우 현재 수석 대표자 ½+1 승인을 통해 투표자의 선택지에서 해당 BP를 블랙리스트에 추가하고 현재 진행 중인 투표를 무효화할 수 있습니다.",
    "exampleDescription4": "BP 선택의 무결성 및 책임성을 극대화합니다.",
    "exampleDescription5": "화이트리스트는 투표자에 의해 고려됩니다",
    "rule1": "참여한 회원 수: 82명",
    "rule2": "선출된 L1 대표자 수: 20명",
    "rule3": "선출된 L2 대표자 수: 5명",
    "rule4": "에덴 회원 ===> 1 점",
    "rule5": "레벨 1 대표자 => 82 / 20 = 4 점",
    "rule6": "레벨 2 대표자 => 82 / 5 = 16 점",
    "whatMyvoteEOS": "마이보트이오스 소개",
    "whatMyvoteEOSDescription": "마이보트이오스는 EOS 메인넷 거버넌스를 위한 자체 규제 기관 이니셔티브입니다. BP 선출에 대한 책임성을 보장하기 위해 녹화 및 공개 가능한 영상 통화에서 3가지 핵심 원칙을 인정하고 동의하는 BP를 필터링합니다.:",
    "whatMyvoteEOSPoint1": "투표 구매를 하지 않습니다.",
    "whatMyvoteEOSPoint2": "운영 및 재무 보고서를 정기적으로 제공합니다.",
    "whatMyvoteEOSPoint3": "2개 이상의 BP(가짜 BP)를 운영하지 않습니다",
    "whatMyvoteEOSDescription2": "모든 프록시는 선택한 BP에 대한 책임을 보장하기 위해 마이보트이오스 화이트리스트를 참조할 수 있습니다. 에덴 스마트 프록시의 투표자는 프록시의 의도와 일치하기 위해 화이티리스트에 등록된 BP들을 선택하도록 권장됩니다.",
    "joinDiscussion": "토론 참여하기:",
    "github": "깃허브:",
    "roadmap": "에덴 스마트 프록시 로드맵",
    "roadmapDescription1": "큰 권력에는 큰 책임이 따르므로, 권력을 가진 사람들은 그들이 대표하는 사람들에 대해 책임을 져야 합니다.",
    "roadmapDescription2": "선출된 BP의 무결성과 책임을 보장하기 위해서는 BP가 자신에게 투표한 투표자에게 직접 보상을 지불하지 않는 것이 중요합니다. BP가 투표에 직접 보상을 지불함으로써 도덕적 해이가 발생합니다. 투표권을 파는 사람들은 본질적으로 보상을 받고 있으므로 EOS 네트워크에 가장 이익이 되지 않는 BP의 저조한 성과 및/또는 다른 행동 등을 간과할 것입니다. 하지만, EOS 보유자가 BP를 선정하기 위해 EOS를 위임하는 대가로 금전적 보상을 받을 것으로 기대하는 선례가 EOS에 있는 것을 부인할 수 없습니다.",
    "roadmapDescription3": "에덴 스마트 프록시는 보상에 대한 투표자들의 요구와 BP의 적합한 책임 사이의 격차를 메우려고 합니다. 이를 위해 EOS 거버넌스는 투표를 받는 사람이 투표자에게 직접 보상하는 대신 투표 행위에 대한 혜택을 줘야 합니다. BP, 투표자, 네트워크 간의 이해관계가 보상 관계의 조정과 함께 적절하게 정렬될 수 있다면 가능합니다.",
    "roadmapDescription4": "만약 EOS 커뮤니티가 에덴 스마트 프록시의 보상 풀로 작은 비율의 인플레이션을 적용하기로 결정한다면, BP는 더 이상 투표자들에게 직접 돈을 지불할 필요가 없을 것입니다.  투표자들은 더 이상 재정적으로만 보상하는 BP에 투표하는 도덕적 위험에 직면하지 않습니다. 대신 투표자들은 에덴의 선거 과정의 정당성과 커뮤니티 구성원들의 책임을 활용하여 BP 선거의 청렴성을 지지한 것에 대해 보상을 받을 것입니다.",
    "roadmapDescription5": "일단 BP들의 인플레이션 분배 변경 결정이 있고 커뮤니티가 이를 지지하면 에덴 스마트 프록시는 네트워크 인플레이션을 통해 스테이킹 보상을 분배할 준비가 되어 있어야 합니다. 따라서 이 제안에 대한 다음 개발 사항은 프록시 위임자를 위한 보상 분배 스마트 계약이 될 것입니다.",
    "roadmapDescription6": "에덴 스마트 프록시가 최적의 결과를 얻기 위해서는:",
    "roadmapList1": "거래소는 BP로부터 리베이트를 받는 대신 고객이 거래소 플랫폼 내에서 직접 APR을 얻을 수 있는 스테이킹 프로그램을 통해 에덴 스마트 프록시에 EOS를 위임할 수 있도록 해야 합니다.",
    "roadmapList2": "BP는 에덴 스마트 프록시에 투표권을 위임한 EOS 보유자에게 보상의 일부를 할당하는 시스템 업그레이드를 구현하는 것을 고려해야 합니다. 만약 BP가 자발적으로 0.25% 블록페이(BPay)를 포기함으로써 그 자금을 에덴 스마트 프록시 풀에 할당하여 BP 선거의 청렴성을 지지한 EOS 보유자에게 보상을 할 수 있다면, 이는 커뮤니티에 의미 있는 신호가 될 것입니다. 또한 이를 통해 BP 순위 21위와 22위 사이의 불필요한 경쟁을 없앨 것입니다.",
    "roadmapList3": "투표자와 EOS 보유자는 이러한 전환 가능성에 대해 전체 네트워크에 무결성과 책임성을 보장하는 동시에 이에 힘을 실어주는 EOS 보유자에게 보상을 주는 것이 훨씬 개선된 이해 관계 조정이라는 인식을 확산시켜야 합니다.",
    "roadmapDescription7": "제네시스(영어권) 에덴 커뮤니티 외에도 에덴 코리아와 에덴 차이나 선거가 준비되고 있다는 점에 주목할 필요가 있습니다. 에덴 스마트 프록시는 각 언어권의 목소리를 반영할 수 있도록 준비해야 합니다. 따라서 에덴 스마트 프록시는 서로 다른 에덴 그룹에서 투표하는 각 회원들이 커뮤니티 크기에 비례하여 투표 가중치를 부여받을 것으로 예상하고 있습니다.",
    "qa": "질의응답",
    "question1": "질문. 투표자들이 보상을 받는다면 투표 판매와는 어떻게 다른가요?",
    "question2": "질문. 이러한 시스템에서 BP가 투표를 구매할까요?",
    "question3": "질문. 이러한 시스템에서 투표자들은 BP에 투표함으로써 보상을 받을 수 있나요?",
    "question4": "질문. 에덴 스마트 프록시에 EOS를 위임하여 보상을 받을 수 있나요?",
    "question5": "질문. 투표 매매가 발생하면 어떻게 되나요? 감시자가 있습니까?",
    "answer1": "답변. 투표 행위 자체에 대한 보상과 투표 판매에 대해 보상 받는 것은 근본적으로 다릅니다.",
    "answer2": "답변. 아니요. 투표 구매를 한다면 에덴 스마트 프록시에서 블랙리스트에 등록 될 수 있습니다.",
    "answer3": "답변. 실제로 BP에게 투표하는 투표자는 에덴 선거 참여자와 에덴 선거를 통해 선출된 대표자입니다. 그들은 투표에 대해 보상을 받지 않습니다. 그러나 에덴 스마트 프록시에 EOS를 위임하여 BP의 무결성과 책임을 지원하는 EOS 보유자는 보상을 받을 것입니다.",
    "answer4": "답변. 네, 하지만 투표 받는 BP가 아닌 네트워크 자체에서 보상을 지급합니다. 이것이 바로 '스마트 프록시'라고 불리는 이유입니다..",
    "answer5": "답변. 에덴 스마트 프록시에는 에덴 수석 대표자들에 의해 임명된 감독관이 배치되며, 이 감독관은 스마트 프록시가 투표하는 BP의 재무 및 기술적 성과를 검토하는 역할을 수행합니다. BP의 부정행위가 적발되고 투표 매수가 발견되면, 보상을 받은 것으로 판명된 투표자와 함께 해당 BP는 블랙리스트에 등록 될 것입니다."
  },
  "vote": {
    "sort": "분류",
    "asc": "오름차순",
    "descending": "내림차순",
    "voteMetaTitle": "투표",
    "mustLogin": "로그인해야 합니다.",
    "beforeVoting": "블록 생산자를 투표하기 전에 프록시에 투표를 위임해야 합니다.",
    "successfulVote": "투표 완료",
    "somethingWrong": "문제가 발생했습니다. 다시 시도해주세요.",
    "loadMore": "더 보기",
    "voteSelected": "선택한 항목에 투표합니다."
  },
  "voters": {
    "votersMetaTitle": "에덴 회원",
    "headTitle": "투표 회원 및 대표자.",
    "lastElection": "지난 선거일",
    "noVoting": "투표하지 않음",
    "voteByOther": "다른 사람에게 투표",
    "voteFor": "투표 BP"
  },
  "home": {
    "homeMetaTitle": "메인",
    "noEdenMember": "에덴 회원만 로그인 가능합니다.",
    "title": {
      "first": "스마트하게 선택하세요",
      "second": "에덴 스마트 프록시를 선택하세요."
    },
    "buttonLabel": "투표 위임",
    "signInLabel": "로그인 및 투표",
    "description": "EdenBPsProxy는 에덴의 프랙탈 민주주의 프로세스를 통해 EOS 보유자들의 목소리를 극대화하는 EOS BP 투표 프록시입니다. 신원을 공개한 에덴 회원만이 BP에 투표하는 프록시에 대해 발언권을 가질 수 있습니다."
  }
}
```

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR do?

- Resolve #151 
- Added some missing translations
- Improved footer style

### CODE REVIEW

**Review by tetogomez — APPROVED** [review]
나에게 잘 보인다

File: public/locales/cn/common.json [hunk 1]
```diff
@@ -86,27 +89,34 @@
     "beforeVoting": "Before voting for Block producers, you must delegate your vote to the proxy.",
     "successfulVote": "Voted Successfully",
     "somethingWrong": "Something went wrong, try again",
-    "loadMore": "Load More",
-    "voteSelected": "Vote Selected"
+    "voteSelected": "Vote Selected",
+    "title1": "Select and vote to whitelist your most trusted 21 BPs.",
+    "title2": "The BP whitelist is sustained by MyvoteEOS",
+    "readMore": "Read more",
+    "selectedAll": "Selecte All"
```
Comment (xavier506) [inline public/locales/cn/common.json:96]: select

File: public/locales/en/common.json [hunk 2]
```diff
@@ -86,27 +89,34 @@
     "beforeVoting": "Before voting for Block producers, you must delegate your vote to the proxy.",
     "successfulVote": "Voted Successfully",
     "somethingWrong": "Something went wrong, try again",
-    "loadMore": "Load More",
-    "voteSelected": "Vote Selected"
+    "voteSelected": "Vote Selected",
+    "title1": "Select and vote to whitelist your most trusted 21 BPs.",
+    "title2": "The BP whitelist is sustained by MyvoteEOS.",
+    "readMore": "Read more",
+    "selectedAll": "Selecte All"
```
Comment (xavier506) [inline public/locales/en/common.json:96]: select

File: public/locales/es/common.json [hunk 3]
```diff
@@ -86,27 +89,34 @@
...
     "title": {
       "first": "Elige sabiamente",
       "second": "Elija Eden Smart Proxy"
     },
-    "buttonLabel": "Voto delegado",
     "signInLabel": "Inicie sesión y vote",
     "description": "EdenBPsProxy es un proxy de votación de EOS BP que maximiza la voz de los titulares individuales de EOS al aprovechar su voz con el proceso de democracia fractal de Eden. Solo los miembros de Edens que revelaron su identidad serán elegibles para opinar sobre el proxy que votos BPs."
+  },
+  "footer": {
+    "communityOwnedPublic": "Un bien público de propiedad comunitaria",
```
Comment (xavier506) [inline public/locales/es/common.json:119]: un bien publico conunitario

**Review by xavier506 — CHANGES_REQUESTED** [review]
spell check

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**